sdk/monitor/monitor-opentelemetry/samples-dev/CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+# Contributing to Azure Monitor OpenTelemetry Samples
```

Comment:
Is this file necessary for a sample project? Do we intend to have users contribute to this sample project?